src/foam/core/Action.js

```diff
@@ -42,7 +42,12 @@ foam.CLASS({
     {
       class: 'String',
       name: 'label',
-      expression: function(name) { return foam.String.labelize(name); }
+      expression: function(name) { return this.displayLabel ? foam.String.labelize(name) : ''; }
+    },
+    {
+      class: 'Boolean',
+      name: 'displayLabel',
```

Comment:
When do we not want to display the label?

Reply:
when we set 'iconFontClass' and only want icon to be displayed

Reply:
Couldn't you just set the 'label' to be ''?

Reply:
The code checks:
if ( this.label ) {
        this.add(this.label$);
      }